src/sentry/event_manager.py

```diff
@@ -1397,7 +1397,7 @@ def _handle_regression(group, event, release):
 
 def _process_existing_aggregate(group, event, data, release):
     date = max(event.datetime, group.last_seen)
-    extra = {"last_seen": date, "score": ScoreClause(group), "data": data["data"]}
```

Comment:
We can remove passing the `ScoreClause` because it is set by the `buffer` service anyway for this case.

https://github.com/getsentry/sentry/blob/master/src/sentry/buffer/base.py#L76-L80